utils/mk-squash-iso.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+help() {
+	cat <<EOF
+mk-squash-iso.sh [options]
+	Turns an sabotage rootfs into an bootable iso.
+	This requires an working isolinux installation on <rootfs>/boot.
+
+	-o <file>
+		Set the output file. Existing files will be overwritten
+		without asking. The file will be an iso image.
+
+	-l <label>
+		Define a custom disk label.
```

Comment:
Its an attribute of the iso9660 filesystem, and will be used to identify the cdrom from within the initramfs. Just specifying "/dev/sr0" wouldn't work when the iso is booted from an USB Stick, which is a feature im planning to support somewhen.

`blkid` usually shows iso9660 labels.